modules/scf/install.sh

```diff
@@ -64,7 +64,15 @@ elif [ "${SCF_OPERATOR}" == "true" ]; then
     --set "global.operator.watchNamespace=scf"
 
     wait_ns cf-operator
-    sleep 10
+
+    info "Wait for cf-operator to be ready"
+    wait_for "kubectl get endpoints -n cf-operator cf-operator-webhook -o name"
+    wait_for "kubectl get crd quarksstatefulsets.quarks.cloudfoundry.org -o name"
+    wait_for "kubectl get crd quarkssecrets.quarks.cloudfoundry.org -o name"
+    wait_for "kubectl get crd quarksjobs.quarks.cloudfoundry.org -o name"
+    wait_for "kubectl get crd boshdeployments.quarks.cloudfoundry.org -o name"
+    sleep 10 # Give extra time for operator to avoid flakyness
```

Comment:
Magic number are bad but if we can't avoid them maybe we should mention in the comment what exactly are we waiting for. This way people will be able to remove the "sleep" in the future by adding more checks like the above.

Reply:
It is there just to avoid random flakiness from the operator (as the comment already suggests) as it might be not ready yet. It's really hard to reproduce locally, the new checks might make the sleep here useless but I would rather avoid removing it right now